templates/news/index.html

```diff
@@ -37,8 +37,8 @@
 <div class="row">
     <h2>IoT</h2>
     <ul>
+        <li><a href="/news/2016/12/19/ubuntu-core-iot/">和Ubuntu Core的一场黑马 - 将物联网的进化革命进行到底&nbsp;&rsaquo;</a></li>
         <li><a href="/news/2016/11/18/ubuntu-core-16-launches">Ubuntu Core 16 为打造安全的物联网奠定基础&nbsp;&rsaquo;</a></li>
-        </li>
```

Comment:
Was this was intentional? I don't see any comments about it in description or commit history.